research-software-directory/RSD-as-a-service
# 877 organisation url

# Fix url for "children" organisations (Research Units)

Closes #877

Changes proposed in this pull request:
* Fix the link to RSD organisation research unit on edit software and edit project pages. The link uses complete rsd path to link to the research units properly.
* Unrelated chores: 
    * removed unused imports spotted while fixing the error 
    * removed frontend flag in api calls
    * applied single component imports for MUI components (build warning)

How to test:
* `make start` to build app and generate data
* login as rsd_admin in order to be able to manipulate existing organisations/software/projects
* navigate to top organisation and create 2 research units at first level and at least one unit at the second level (child of a child organisation)
* navigate to first software from the overview and edit it. Add new research units you just created. Verify that the link opens the organisation page with that research unit. Verify that other organisation links also are valid.
* navigate to first project from the overview and edit it. Add new research units you just created. Verify that the link opens the organisation page with that research unit. Verify that other organisation links also are valid.


PR Checklist:
